scss/bootstrap/overrides/style/_buttons.scss

```diff
@@ -103,6 +103,18 @@
   }
 }
 
+.btn-default-outline {
+  background-color: transparent !important;
```

Comment:
I tried that, but using it still needed me to override few things
1. `border-color` is `#C2CFD8` in that case, which should be white.
Other differences in the GIF
![1](https://user-images.githubusercontent.com/5058867/27787144-8012b41e-6001-11e7-9798-01c3d74a8896.gif)